Basket-Donation-Payments/Basket-Donation-Payments/Models/APIManager.m

```diff
@@ -140,4 +139,41 @@ + (void) newNonprofitConnectedAccountWithEmail:(NSString*)email withAuthorizatio
     [task resume];
 }
 
++ (void)createTransfersWithAmount:(NSNumber*)amount toConnectedStripeAccs:(NSArray<NSString*>*)connectedStripeAccs withSourceTxId:(NSString*)sourceTxId withBlock:(void (^)(NSError *, NSString *))completion {
+    NSInteger numberOfNonprofits = connectedStripeAccs.count;
+    NSInteger amountPerNonprofit = [amount intValue] / numberOfNonprofits;
+    for (NSString *connectedStripeId in connectedStripeAccs) {
```

Comment:
Here's when you would normally use a `NSOperationQueue`, else the completion block will be called every time a single transaction is completed. You can also use recursion if that's simpler, and only call the completion block at the final recursive step when there are no more pending transfers.